includes/widgets/common.php

```diff
@@ -1098,6 +1098,28 @@ protected function register_controls() {
 				]
 			);
 
+			$this->add_responsive_control(
+				"_transform_perspective_effect{$tab}",
+				[
+					'label' => __( 'Perspective', 'elementor' ),
+					'type' => Controls_Manager::SLIDER,
+					'range' => [
+						'px' => [
+							'min' => 0,
+							'max' => 1000,
+						],
+					],
+					'condition' => [
+						"_transform_rotate_popover{$tab}!" => '',
+						"_transform_rotate_3d{$tab}!" => '',
+					],
+					'selectors' => [
+						'{{WRAPPER}}' => '--e-transform-perspective: {{SIZE}}px',
```

Comment:
I see that all other variables are targeted to:
`"{{WRAPPER}} > .elementor-widget-container{$state}"`
Is it on purpose that this one is directly on the wrapper?

Reply:
Fixed